app/account_linking.py

```diff
@@ -38,6 +51,38 @@ class LinkResult:
     strategy: str
 
 
+def send_double_subscription_if_needed(partner_user: PartnerUser):
+    sub = partner_user.user.get_active_subscription()
+    if isinstance(sub, CoinbaseSubscription):
+        channel = "Coinbase"
```

Comment:
Coinbase subscription isn't renewable so users can't cancel it.

Reply:
I'll remove it.